### ISSUE TITLE
Backup tool

### ISSUE THREAD

**Issue by hauxir** [hauxir/OpenBazaar-Server#1]
Building a backup tool to export all of your store data into a single backup file that can be imported into a fresh install.

**Comment by marcjamot** [comment]
First version done and added to API, checking with developer at OpenBazaar to make sure it's how they want it.

**Comment by hauxir** [comment]
How is this going? Have you looked at the comments from the devs?

**Comment by marcjamot** [comment]
Yeah. Looking into how sqlite backup api works with python and how the restored data can collide with the existing data. Also to build the table structure automatically which could be done with the "sqlite_master" table and "pragma table_info".

**Comment by hauxir** [comment]
So how's this going now? It needs more test coverage?
Maybe @gpseiridis can look at that?
